src/ripple/overlay/impl/OverlayImpl.cpp

```diff
@@ -695,6 +696,10 @@ OverlayImpl::onManifests (
                 app_.validators(),
                 journal);
 
+            if (result == ManifestDisposition::accepted ||
+                    result == ManifestDisposition::untrusted)
+                app_.getOPs().pubManifest (std::move(*make_Manifest(serialized)));
```

Comment:
`std::move` is not needed here, `*make_Manifest` will already be an rvalue ref. Plus the parameter to `pubManifest` is a `Manifest const&`, so it's not going to take advantage of move semantics anyway.